manifests/base/jupyter-pytorch-notebook-imagestream.yaml

```diff
@@ -17,23 +17,34 @@ spec:
   # N Version of the image
   - annotations:
       opendatahub.io/notebook-software: '[{"name":"CUDA","version":"11.8"},{"name":"Python","version":"v3.9"},{"name":"PyTorch","version":"1.13"}]'
-      opendatahub.io/notebook-python-dependencies: '[{"name":"PyTorch","version":"1.13"},{"name":"Tensorboard","version":"2.11"},{"name":"Boto3","version":"1.26"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.24"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.2"},{"name":"Scipy","version":"1.10"},{"name":"Elyra","version":"3.15"}]'
+      opendatahub.io/notebook-python-dependencies: '[{"name":"PyTorch","version":"2.0"},{"name":"Tensorboard","version":"2.13"},{"name":"Boto3","version":"1.28"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.24"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.3"},{"name":"Scipy","version":"1.11"},{"name":"Elyra","version":"3.15"},{"name":"PyMongo","version":"4.5"},{"name":"pyodbc","version":"4.0"}, {"name":"Codeflare-SDK","version":"0.8.0"}, {"name":"sklearn-onnx","version":"1.15"}, {"name":"Psycopg2","version":"2.9"}, {"name":"MySQL Connector/Python","version":"8.0"}]]'
```

Comment:
We need to keep only the x.y on the versions so the Codeflare-SDK 0.8.0 should change to Codeflare-SDK 0.8.
The Psycopg2 2.9 don't exist instead there is the psycopg 3.1